README.md

```diff
@@ -122,10 +122,19 @@ chameleon.create({
 });
 ```
 
+### Options as JS or JSON config file
+
+The above Javascript options can be persisted as either JS or a JSON configuration files. The files must be named chameleon.config.js or chameleon.config.json.
```

Comment:
@harterSe Please use singular "file" instead of "files", since having both JS and JSON doesn't make sense. Btw, what would happen if multiple files are found?

Reply:
The first file found would be used. The order in the array is used for prioritization.

```
// if js file is found json file is ignored
const configFileNames = ['chameleon.config.js', 'chameleon.config.json'];
```